.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+os: linux
+sudo: false
+
+python:
+  - "3.5"
```

Comment:
Why are we using python 3.5 when 3.7 is out and the most current version? Or at least 3.6 since just about everything supports that currently?

Reply:
@nathankurt No specific reason. That's what ubuntu 16.04 ships with so that's what I used. I checked python change log and there's no real big issue that was fixed that warranted python upgrade. Updating to 3.6 would however create some problems while deploying because that's digital ocean droplets use 16.04

Reply:
hmmmmmmmmmm, since we aren't using aws and have our own droplet, why not just install it on the droplet?

Reply:
lol no need to censor yourself. But server config with 3.6 would be a pain? If there aren't enough major improvements to justify the time, then that's fine I guess. But if you think upgrading to 3.7 it might fix an issue or make something eaiser, I'd do that. I'll leave it in your hands though.

Reply:
I will leave that to be a thing to be done once we get everything up.